submissions/sprint_c/app.py

```diff
@@ -0,0 +1,32 @@
+from __init__ import app
+from flask import jsonify
+from helpers import EpithetGenerator
+
+
+@app.route('/')
+def generate_epithets():
+    single_epithet = EpithetGenerator.display_single_epithet()
+    return jsonify({"epithet": single_epithet})
+
+
+@app.route('/vocabulary')
+def vocabulary():
+    vocab_dataset = EpithetGenerator.display_vocab_dataset()
+    return jsonify({"vocabulary": vocab_dataset})
+
+
+@app.route('/epithets/<quantity>')
+def quantity(quantity):
+    epithets = EpithetGenerator.get_quantity_of_epithets(int(quantity))
```

Comment:
You can avoid typecasting yourself using variable rules
http://flask.pocoo.org/docs/1.0/quickstart/#variable-rules